src/views/schemaBrowser/schemaBrowser.ts

```diff
@@ -431,10 +445,22 @@ export default class schemaBrowser {
       const contextValue = element.contextValue;
 
       if (element instanceof Schema) {
-        items = getSchemaItems(element.schema);
+
+        let filterValue = this.filters[element.schema];
+        if (filterValue) {
+          filterValue = Statement.noQuotes(Statement.delimName(filterValue, true));
+          const filteredObjects = await Schemas.getObjects(element.schema, AllSQLTypes, {filter: filterValue});
```

Comment:
For a schema with a delimited name, `element.schema` includes the delimiting quotes.  Those need to be stripped off when passed into `getObjects`.

Reply:
@davecharron that is what `noQuotes` does, unless you are referring to elsewhere in the code?

Reply:
On line 451 it's applying it to the filter value, not the schema name.

Reply:
Fixed in latest commit. Took me a while to catch on.